packages/firebase/index.d.ts

```diff
@@ -5472,6 +5472,52 @@ declare namespace firebase.storage {
      *     including if the object did not exist.
      */
     updateMetadata(metadata: firebase.storage.SettableMetadata): Promise<any>;
+    /**
+     * List items and folders (prefixes) within this directory.
+     * "/" is treated as a path delimiter. Firebase storage does not support invalid object path that ends with "/" or contains two consecutive "//".
+     * All invalid objects in GCS will be filtered.
+     *
+     * @param {!options.maxResults} If set, limits the total number of prefixes and items to return..
```

Comment:
I know that the rest of the file uses these exclamation marks, but they are only used in JSDoc to describe the type of the argument. Here (and everywhere else in the Storage section), they seem to be used to define the argument name. That doesn't seem correct. 

You should also drop the curly braces for the argument names.